packages/components/src/Actions/Action/index.js

```diff
@@ -1,4 +1,3 @@
-import Action, { wrapOnClick } from './Action.component';
+import Action from './Action.component';
```

Comment:
`wrapOnClick` is imported by `ActionButton` and `ActionDropdown`. But those 2 components are imported by Action too, making a cyclic dependency.

I moved `wrapOnClick` in its own file, that is imported by the 2 components.